app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :posts, class_name: 'Post', foreign_key: :post_id
```

Comment:
- [ ] Here it should be `belongs_to :post`, there is no need to define `class_name & foreign_key`.